mocks/controllers/InformationController.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery v2.14.0. DO NOT EDIT.
```

Comment:
In commit "update mockery version" missing mockery dependance update.

Reply:
Please add this commit as first commit

Reply:
In commit "update mockery version" missing mockery dependance update: I don't understand what you mean exactly
Please add this commit as first commit: done